src/main/java/logisticspipes/modules/ModuleCrafter.java

```diff
@@ -788,21 +747,19 @@ public IRouter getSatelliteRouter(int x) {
         if (x == -1) {
             for (final PipeItemsSatelliteLogistics satellite : PipeItemsSatelliteLogistics.AllSatellites) {
                 if (satellite.satelliteId == satelliteId) {
-                    CoreRoutedPipe satPipe = satellite;
-                    if (satPipe == null || satPipe.stillNeedReplace() || satPipe.getRouter() == null) {
+                    if (satellite == null || satellite.stillNeedReplace() || satellite.getRouter() == null) {
```

Comment:
why checking for null is after the checking for id?

Reply:
it's the original code lol